test/mobile/custom_build/build.sh

```diff
@@ -49,6 +49,7 @@ generate_op_dependency_graph() {
       ANALYZE_TORCH=1 \
       "${SRC_ROOT}/tools/code_analyzer/build.sh" -closure=false
   fi
+  cat "$OP_DEPENDENCY"
```

Comment:
This will print ~7k lines to the stdout.. Maybe make this line more visible? https://github.com/pytorch/pytorch/blob/master/tools/code_analyzer/run_analyzer.sh#L20

Reply:
but it's fine if the goal is to see the results from custom build CI job page without having to run it locally.